### PR TITLE
Latest changes to nav and toolbar

### PR DESCRIPTION
**Pull request by clairek20**
## Pull request -
Latest changes to nav and toolbar:
Change the chevron icon direction when dropdown is open and closed
Fix hover spacing. The hover was only hovering over 3/4 of the panel
Remove highlighting on selected dropdowns
Increase icon size to 20px
Increase the menu item size to 32px.

### Affected issues

- Resolves 

### Proposed changes

Change the chevron icon direction when dropdown is open and closed
Fix hover spacing. The hover was only hovering over 3/4 of the panel
Remove highlighting on selected dropdowns
Increase icon size to 20px
Increase the menu item size to 32px.

### CODE REVIEW

**Comment by clairek20** [comment]
Hey @Deriohig 

Could you look into reviewing this today, please? Hoping to grab the changes this evening if possible for sprint end tomorrow.

Thanks, Claire.

File: src/components/Nav/_mixins.scss [hunk 1]
```diff
@@ -194,7 +184,8 @@ $nav__icon__spacing: $carbon--spacing-05 + $nav__spacing;
 
         position: relative;
         display: flex;
-        padding: $carbon--spacing-02;
+        height: 32px;
```
Comment (Deriohig) [inline src/components/Nav/_mixins.scss:187]: this value needs to be $spacing-07

File: src/components/Nav/_mixins.scss [hunk 2]
```diff
@@ -194,7 +184,8 @@ $nav__icon__spacing: $carbon--spacing-05 + $nav__spacing;
...
+        padding: 0.3125rem $carbon--spacing-02 $carbon--spacing-02 2.3rem;
```
Comment (Deriohig) [inline src/components/Nav/_mixins.scss:188]: is there a [spacing variable](https://www.carbondesignsystem.com/guidelines/spacing/overview/) you can use here?

File: src/components/Toolbar/_mixins.scss [hunk 3]
```diff
@@ -199,13 +199,12 @@ $toolbar__panel__transition__timing-function: $timing-function;
   }
 
   &__nav__item__icon {
-    width: carbon--mini-units($count: 2);
-    height: carbon--mini-units($count: 2);
+    width: carbon--mini-units($count: 2.5);
+    height: carbon--mini-units($count: 2.5);
+    margin-left: -35px;
```
Comment (Deriohig) [inline src/components/Toolbar/_mixins.scss:204]: whats the negative margin for?

File: src/components/Nav/_mixins.scss (hunk 2 above)
Comment (clairek20) [reply src/components/Nav/_mixins.scss]: Done!

File: src/components/Nav/_mixins.scss (hunk 1 above)
Comment (clairek20) [reply src/components/Nav/_mixins.scss]: Done!

File: src/components/Toolbar/_mixins.scss (hunk 3 above)
Comment (clairek20) [reply src/components/Toolbar/_mixins.scss]: I need to use a negative margin to draw the element closer to the left neighbour to align it with items without icons

**Review by Deriohig — CHANGES_REQUESTED** [review]
image

You have lost alignment on the sections title's and nav items. Most likely from line 207 in `Toolbar/_mixins.scss`

**Review by Deriohig — APPROVED** [review]
Thanks for the changes 👍

**Pull request merged**